sunpy/data/test/__init__.py

```diff
@@ -103,16 +107,15 @@ def write_image_file_from_header_file(header_file, fits_directory):
     return fits_file
 
 
-def get_dummy_map_from_header(filename):
+def get_dummy_map_from_header(filepath):
     """
     Generate a dummy `~sunpy.map.Map` from header-only test data.
 
     The "image" will be random numbers with the correct shape
     as specified by the header.
     """
-    filepath = get_test_filepath(filename)
```

Comment:
Due to the above API change, I think we should make this function use the file path instead of having to add more args to this.